os_info/Cargo.toml

```diff
@@ -37,6 +37,9 @@ windows-sys = { version = "0.52", features = [
     "Win32_UI_WindowsAndMessaging",
 ] }
 
+[target.'cfg(target_os = "macos")'.dependencies]
+plist = "1.5.1"
```

Comment:
Instead of an XML-parser crate, parsing via native APIs (or querying those for the version) would also be possible.